webpack/assets/javascripts/react_app/routes/common/callOnMount/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const callOnMount = callback => WrappedComponent =>
```

Comment:
IMO the `withRenderHandler` can run `componentDidMount` internally.
I believe once it has done, this HOC will not be needed anymore.

I don't see a benefit using it instead `useEffect` or `componentDidMount`. I actually think it takes for developers more time to understand how things works when using HOCs as they can be very confusing.

Reply:
we dont need this, we can use [lifecycle](https://github.com/acdlite/recompose/blob/master/docs/API.md#lifecycle), I wrote this for things to be more understandable,